docs/usage/dataclasses.md

```diff
@@ -377,3 +377,34 @@ print(pydantic_core.to_json(user, indent=4).decode())
 }
 """
 ```
+## Attribute copies
+
+As described earlier, when constructing classes with data attributes, Pydantic copies the the attribute in order to efficiently iterate over its elements for validation.
+
+In this example, note that the ID of the list changes after the class is constructed because it has been copied for validation.
+
+```py
+from typing import List
+
+from pydantic import BaseModel
+
+
+class C1:
```

Comment:
Was this supposed to be a dataclass? There's no reference to dataclasses in this section. 🤔

Reply:
You are totally correct. Moving this to models